site/topics/control-logic/instruction_register_and_control_logic.dig

```diff
@@ -500,7 +500,7 @@
           <data>8004,2802,0,0,8004,2802,9000,2008,8004,2802,1008,0,8004,2802,9000
```

Comment:
The tests and signals were updated. Order of emphasized signals were swapped. 
![image](https://github.com/user-attachments/assets/35cf5116-fda2-4e28-aec1-9a018ab51912)